### PR TITLE
{Release} Upgrade to Azure CLI 2.37.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,190 @@
...
+* `az bicep decompile`: Add new parameter `--force` to allow overwriting existing Bicep files (#22423)
+* `az resource wait`: Fix `--created` keeps waiting even when `az resource show` returns "provisioningState": "Succeeded" (#22254)
+
+**ARO**
+
+* `az aro create`: Add support for FIPS modules, host encryption, and disk encryption for master and worker nodes (#22320)
+
+**Azure/azure-cli**
+
+* [Service Connector] `--secret name=xx keyVaultSecretUri=xx`: Add new parameter key value pair to support password from KeyVault (#22319)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:56]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,190 @@
...
+* Add multiple user authentication (MUA) support for critical operations: `az backup vault backup-properties set`/`az backup item set-policy`/`az backup policy set`/`az backup protection disable` (#22472)
+* Add `--tenant-id` parameter in critical commands: `az backup vault backup-properties set`/`az backup item set-policy`/`az backup policy set`/`az backup protection disable`/`az backup vault resource-guard-mapping` for cross-tenant scenario (#22472)
+
+**Compute**
+
+* `az vm image list`: Add new server version aliases `Win2022AzureEditionCore` for offline list (#22193)
+* `az vm update`: Add additional license type SLES for `--license-type` (#22336)
+* `az vmss create`: Support enabling single placement group for Flexible VMSS (#22291)
+* `az disk create/update`: Add new parameter `--data-access-auth-mode` to support data access authentication mode (#22175)
+* `az sig show`: Add new parameters `--sharing-groups` to support query shared gallery group (#22371)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:70]:
```suggestion
* `az sig show`: Add new parameter `--sharing-groups` to support query shared gallery group (#22371)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,190 @@
...
+* Migrate adal graph sdk to msgraph sdk in HDInsight module (#22503)
+
+**Identity**
+
+* `az identity list-resources`: Add new command to support list the associated resources for identity (#22519)
+
+**IoT**
+
+* `az iot dps policy` and `az iot dps linked-hub`: Fix DPS state updating (#22259)
+* Test updates for beta branch (#22262)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,190 @@
...
+**Key Vault**
+
+* Fix#22457: `az keyvault key decrypt/encrypt`: Fix returning bytes for `--output tsv` (#22464)
+
+**Monitor**
+
+* [BREAKING CHANGE] `az monitor alert`: deprecate whole command group, please use `monitor metrics alert` (#22507)
+* [BREAKING CHANGE] `az monitor autoscale-settings`: deprecate whole command group, please use `az monitor autoscale` (#22507)
+* [BREAKING CHANGE] `az monitor activity-log list`: deprecate parameter `--filters`. (#22507)
+* [BREAKING CHANGE] `az monitor activity-log list`: deprecate parameter flag `--resource-provider`, please use `--namespace` (#22507)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:108]:
```suggestion
* [BREAKING CHANGE] `az monitor alert`: Deprecate whole command group, please use `monitor metrics alert` (#22507)
* [BREAKING CHANGE] `az monitor autoscale-settings`: Deprecate whole command group, please use `az monitor autoscale` (#22507)
* [BREAKING CHANGE] `az monitor activity-log list`: Deprecate parameter `--filters`. (#22507)
* [BREAKING CHANGE] `az monitor activity-log list`: Deprecate parameter flag `--resource-provider`, please use `--namespace` (#22507)
```

**Comment by yonzhan** [comment]
Release Azure CLI 2.37.0

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,190 @@
...
+* `az servicebus namespace create/update`: Add `--disable-local-auth` to enable or disable SAS authentication (#19741)
+* `az servicebus namespace private-endpoint-connection/private-link-resource`: New command groups (#19741)
+
+**Service Connector**
+
+* [BREAKING CHANGE] `az containerapp connection create`: Default client_type changed to `none` (#22311)
+* `az containerapp connection`: Add new command group to support container app connection (#22290)
+* `az containerapp connection create`: Add `--container` parameter in interactive mode (#22311)
+* `az spring connection`: Add support for `az sping-cloud` renaming (#22356)
+
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:162]:
```suggestion
Add new parameter key value pair to support password from KeyVault (#22319)

```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,186 @@
...
+* [BREAKING CHANGE] `az az/role`: Migrate the underlying API of `az ad` and `az role` from AD Graph API to Microsoft Graph API. For more details, see [Microsoft Graph migration](https://docs.microsoft.com/en-us/cli/azure/microsoft-graph-migration) (#22432)
+
+**Security**
+
+* `az security alerts-suppression-rule`: Add alerts suppression rules to security module (#22014)
+
+**Service Bus**
+
+* `az servicebus queue update`: Fix message time to live (#22218)
+* : `az servicebus queue`: Adding ReceiveDisabled to --status (#22460)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:147]:
```suggestion
* `az servicebus queue`: Adding ReceiveDisabled to --status (#22460)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,186 @@
...
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst]:
```suggestion
* `az servicebus queue`: Add `ReceiveDisabled` to `--status` (#22460)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,190 @@
...
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst]:
```suggestion
* `az servicebus queue`: Adding ReceiveDisabled to `--status` (#22460)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,186 @@
...
+* [BREAKING CHANGE] `az storage share show`: Remove contentLength, hasImmutabilityPolicy and hasLegalHold from the output result (#22215)
+* [BREAKING CHANGE] `az storage blob snapshot`: Now only returns version info instead of all blob properties (#22309)
+* Fix #21819: `az storage fs directory`: Add new command `generate-sas` (#22152)
+* `az storage account show-connection-string`: Append endpoints by default (#22280)
+* Fix #22236: `az storage entity insert`: Fix `--if-exists fail` not working (#22334)
+* `az storage copy`: Fix `--exclude-path` TypeError (#22367)
+* `az storage blob download`: Allow downloading to stdout for pipe support (#22317)
+* Fix #22209: `az storage entity insert`: Fix `Edm.Boolean` not working (#22483)
+* `az storage directory/file list`: Add `--exclude-extended-info` to exclude some properties info from response, default to `False` (#22490)
+* Fix #21781 `az storage blob upload/download`: Progress fix (#22504)
```
Comment (necusjz) [inline src/azure-cli/HISTORY.rst:182]:
```suggestion
* Fix #21781: `az storage blob upload/download`: Progress fix (#22504)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,190 @@
...
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:182]:
```suggestion
* Fix #21781: `az storage blob upload/download`: Progress fix (#22504)
```

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,186 @@
...
+* `az cosmosdb sql container update`: Fix bug to accept analyticalStorageTTL arg (#22132)
+
+**Event Hubs**
+
+* `az eventhubs namespace schema-registry`: Add cmdlets for schema registry (#22100)
+
+**HDInsight**
+
+* Migrate adal graph sdk to msgraph sdk in HDInsight module (#22503)
+
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:80]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 12]
```diff
@@ -3,6 +3,186 @@
...
+* `az iot dps policy` and `az iot dps linked-hub`: Fix DPS state updating (#22259)
+* `az iot central app private-link-resource list`: Add a new command to support listing private link resources (#22273)
+* `az iot central app private-endpoint-connection show`: Add a new command to support showing details of a private endpoint connection of the IoT Central app (#22273)
+* `az iot central app private-endpoint-connection approve`: Add a new command to support approving a private endpoint connection for the IoT Central app (#22273)
+* `az iot central app private-endpoint-connection reject`: Add a new command to support rejecting a private endpoint connection for the IoT Central app (#22273)
+* `az iot central app private-endpoint-connection delete`: Add a new command to support deleting a private endpoint connection for the IoT Central app (#22273)
+
+**Key Vault**
+
+* Fix#22457: `az keyvault key decrypt/encrypt`: Fix returning bytes for `--output tsv` (#22464)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
* Fix #22457: `az keyvault key decrypt/encrypt`: Fix returning bytes for `--output tsv` (#22464)
```

File: src/azure-cli/HISTORY.rst [hunk 13]
```diff
@@ -3,6 +3,186 @@
...
+* [BREAKING CHANGE] `az az/role`: Migrate the underlying API of `az ad` and `az role` from AD Graph API to Microsoft Graph API. For more details, see [Microsoft Graph migration](https://docs.microsoft.com/en-us/cli/azure/microsoft-graph-migration) (#22432)
+
+**Security**
+
+* `az security alerts-suppression-rule`: Add alerts suppression rules to security module (#22014)
+
+**Service Bus**
+
+* `az servicebus queue update`: Fix message time to live (#22218)
+* `az servicebus queue`: Adding ReceiveDisabled to --status (#22460)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:143]:
```suggestion
* `az servicebus queue`: Add ReceiveDisabled to --status (#22460)
```

File: src/azure-cli/HISTORY.rst [hunk 14]
```diff
@@ -3,6 +3,182 @@
...
+
+* `az network watcher connection monitor create`: Change for using user-provided workspace-ids even if output-type is missing (#22156)
+* `az network dns zone export`: Support traffic manager resources (#22205)
+* Private link add `Microsoft.Kusto/clusters` provider (#22178)
+* `az network lb create`: Add warnings for default SKU (#22339)
+* `az network lb address-pool`: Support connection draining on load balancer (#22508)
+* `az network application-gateway`: Add `settings`, `listener` and `routing-rule` command groups (#22489)
+* `az network application-gateway create`: Add parameter `--priority` (#22489)
+* `az network application-gateway probe`: Add parameter `--host-name-from-settings` (#22489)
+
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:117]:
```suggestion
* [BREAKING CHANGE] `az network vnet peering`: Deprecate parameter flag `--remote-vnet-id` (#22522)

```

**Review by zhoxing-ms — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**